js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -1,6 +1,6 @@
 /* jshint browser: true */
 /* jshint unused: false */
-/* global Backbone, $, _, window, templateEngine, arangoHelper, GraphViewerUI, Joi, frontendConfig */
+/* global Backbone, $, _, window, templateEngine, arangoHelper, GraphViewerUI, Joi, frontendConfig, localStorage */
```

Comment:
instead of defining globals like this for stuff which already exists on window, we can just do `window.localStorage`

Reply:
Resolved within https://github.com/arangodb/arangodb/pull/18199/commits/60e53273452d4e245f570f5c3c957f9f99d78ab7